docs/UserGuide.md

```diff
@@ -202,18 +202,35 @@ _Details coming soon ..._
 ## Known issues
 
 1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
-
+2. **Restrictions on the valid user input of `Faculty` values**: For the current version, a valid user for `Faculty` with prefix `f/` can only match exactly the same string as suggested below. Cases can be ignored though.
```

Comment:
should `For the current version, a valid user for `Faculty`` be `For the current version, a valid user **input** for `Faculty``?